tests/FakeItEasy.IntegrationTests/TypeCatalogueTests.cs

```diff
@@ -95,7 +101,7 @@ public void Should_warn_of_bad_assembly_files()
                 }
 
                 writer.Flush();
-                actualMessage = writer.Encoding.GetString(stream.GetBuffer());
+                actualMessage = writer.Encoding.GetString(stream.ToArray());

```

Comment:
`ToArray` is more correct than `GetBuffer` in this case anyway, so this change also fixes a possible bug :wink: